google/cloud/completion_queue.h

```diff
@@ -32,7 +32,7 @@ class CompletionQueue;
 
 namespace internal {
 std::shared_ptr<CompletionQueueImpl> GetCompletionQueueImpl(
-    CompletionQueue& cq);
+    CompletionQueue const& cq);
```

Comment:
as you probably noticed, that broke the `clang-tidy` checks in some bigtable tests, blegh.  Maybe a good reason to split the PR?  Just a thought.

Reply:
As long as there are no larger fixes, I think it would be simpler to keep this together as one.